Source/Sandbox/Program.cs

```diff
@@ -0,0 +1,44 @@
+﻿using PhantomJs.NetCore;
+using System;
+using System.IO;
+
+namespace Sandbox
+{
+  public class ExampleProgram
+  {
+    public static void Main(string[] args)
+    {
+      var currentDirectory = Directory.GetCurrentDirectory();
+      var generator = new PdfGenerator();
+
+      var htmlToConvert =
+@"
+<!DOCTYPE html>
+<html>
+<head>
+<style>
+// Had to use this to fix the font sizing problem in linux.
```

Comment:
Lets use /* */ instead of // as CSS does not suppot //. This probably does not crash, but I don't know if it might.

Reply:
Okay, gonna change this too